### PR TITLE
Add Metric Store Group as contributers

### PR DESCRIPTION
**Pull request by jbooherl**
We have been maintaining the metric store release and believe that we meet criteria based on support interactions in  #op-apps-assist-tas-metric-store

### CODE REVIEW

File: toc/working-groups/app-runtime-platform.md [hunk 1]
```diff
@@ -278,7 +278,7 @@ areas:
   - cloudfoundry/loggregator-release
   - cloudfoundry/loggregator-tools
   - cloudfoundry/metric-store-ci
-  - cloudfoundry/metric-store-dotfiles
+  - `
```
[the remark this answers is not in the document]
Comment (jbooherl) [reply toc/working-groups/app-runtime-platform.md]: Whoops. Updated and force pushed. I added the repo to the metric-store groups so we can archive it

File: toc/working-groups/app-runtime-platform.md [hunk 2]
```diff
@@ -287,6 +287,22 @@ areas:
...
+  - name: Hovhannes Manukyan
+    github: hmanukyanVMw
+  - name: Gevorg Gevorgyan
+    github: gggevorgyan
+  - name: Meri Yeghiazaryan
+    github: myeghiazarya
+  - name: svcboteos
+    github: svcboteos
+  repositories:
+  - cloudfoundry/metric-store-ci
```
[the remark this answers is not in the document]
Comment (jbooherl) [reply toc/working-groups/app-runtime-platform.md:300]: Removed the repos from the Metrics and Logging teams.

File: toc/working-groups/app-runtime-platform.md [hunk 3]
```diff
@@ -287,6 +287,22 @@ areas:
...
+  - name: Jeanette Booher
+    github: jbooherl
+  - name: Hovhannes Manukyan
+    github: hmanukyanVMw
+  - name: Gevorg Gevorgyan
+    github: gggevorgyan
+  - name: Meri Yeghiazaryan
+    github: myeghiazarya
+  - name: svcboteos
+    github: svcboteos
```
[the remark this answers is not in the document]
Comment (jbooherl) [reply toc/working-groups/app-runtime-platform.md:298]: Done

File: toc/working-groups/app-runtime-platform.md [hunk 4]
```diff
@@ -287,6 +287,22 @@ areas:
   - cloudfoundry/statsd-injector-release
   - cloudfoundry/system-metrics-scraper-release
 
+- name: Metric Store
+  approvers:
+  - name: Jeanette Booher
+    github: jbooherl
```
[the remark this answers is not in the document]
Comment (jbooherl) [reply toc/working-groups/app-runtime-platform.md:290]: Done! https://github.com/cloudfoundry/community/pull/650

**Comment by jbooherl** [comment]
> It is my understanding that this will make the working group yaml match the state of the world and that these contributors have been working on these projects for a long time. However, I am less familiar with this group and want to make sure that everyone meets the same standard for approver as described [here](https://github.com/cloudfoundry/community/blob/main/toc/ROLES.md#approver). Please provide details about how long each person has been working on these projects and links to their contributions.

I have been working as the tech lead on the project since September of 2022. 
Metric Store is being maintained but there have not been a lot of sweeping changes. It is used as the backend for App Metrics which is still in use by customers and we do regular tile releases of the product.
Hovhannes is the anchor of the project. I believe his support ticket work in #op-apps-assist-tas-metric-store and possibly PRs qualify him. 
Gevorg has also been supporting metric store and making changes to the [metric store repo](https://github.com/cloudfoundry/metric-store-release/commits?author=gggevorgyan)
Meri has just returned from parental leave and prior to that also committed to [metric store](https://github.com/cloudfoundry/metric-store-release/commits?author=myeghiazarya)
> 
> Here are several examples of people showing their work to apply as an approver
> 
> * [An example of the easiest way to link commits. ](https://github.com/cloudfoundry/community/pull/644)
> * [ An example of a more thorough list](https://github.com/cloudfoundry/community/pull/628)
> 
> ❓ @Benjamintf1 - This PR will be adding ownership to the metric-store repos, which are currently in your area. Does this split make sense to you?

File: toc/working-groups/app-runtime-platform.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (stephanme) [reply toc/working-groups/app-runtime-platform.md:298]: Bots should be added as bots and not as approvers. Bots can be configured on WG level and on WG area level.
Bots don't need to be added as contributors. All bots (and approvers, reviewers, leads) get added to the cloudfoundry org automatically.

See [rfc-0014-github-teams-and-access.md](https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0014-github-teams-and-access.md) for the possible teams and permissions. 
The [Service Management WG](https://github.com/cloudfoundry/community/blob/main/toc/working-groups/service-management.md) contains examples for WG bot and WG area bot configuration.

**Review by stephanme — CHANGES_REQUESTED** [review]

File: toc/working-groups/app-runtime-platform.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jbooherl) [reply toc/working-groups/app-runtime-platform.md]: Moved the bot to the upper WG section

**Comment by Benjamintf1** [comment]
@ameowlia yeah, we havn't really been ever maintaining the metric-store repos.

File: toc/working-groups/app-runtime-platform.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ameowlia) [reply toc/working-groups/app-runtime-platform.md]: Oops! Good catch @stephanme. I didn't realize it was a bot 🤦‍♀️

**Review by ameowlia — APPROVED** [review]

**Pull request merged**